src/modules/always_on_top/dllmain.cpp

```diff
@@ -0,0 +1,265 @@
+#include "pch.h"
+#include <interface/powertoy_module_interface.h>
+#include <interface/powertoy_system_menu.h>
+#include <common/settings_objects.h>
+#include <set>
+#include "trace.h"
+#include "resource.h"
+
+extern "C" IMAGE_DOS_HEADER __ImageBase;
+
+BOOL APIENTRY DllMain(HMODULE hModule, DWORD  ul_reason_for_call, LPVOID lpReserved) {
+  switch (ul_reason_for_call) {
+  case DLL_PROCESS_ATTACH:
+    Trace::RegisterProvider();
+    break;
+  case DLL_THREAD_ATTACH:
+  case DLL_THREAD_DETACH:
+    break;
+  case DLL_PROCESS_DETACH:
+    Trace::UnregisterProvider();
+    break;
+  }
+  return TRUE;
+}
+
+const static wchar_t* MODULE_NAME = L"AlwaysOnTop";
+const static wchar_t* MODULE_DESC = L"";
+const static wchar_t* HOTKEY_NAME = L"AlwaysOnTop_HotKey";
+
+struct AllwaysOnTopSettings {
+  // Default hotkey WIN + ALT + T
+  PowerToysSettings::HotkeyObject editorHotkey =
+    PowerToysSettings::HotkeyObject::from_settings(true, false, true, false, 0x54, L"T"); // ASCII code for T 0x54
+};
+
+namespace {
+  const std::wstring KMenuItemName = L"Always on top";
```

Comment:
Since this is the name of the module, and not just a Windows command, let's use `Always On Top` here as well.